pysatMissions/instruments/methods/orbits.py

```diff
@@ -0,0 +1,141 @@
+"""Methods to convert orbital information for propagators."""
+
+import numpy as np
+import warnings
+
+
+def _check_orbital_params(kwargs=None):
+    """Check that a complete set of unconflicted orbital parameters exist.
+
+    Parameters
+    ----------
+    kwargs : dict or NoneType
+        Dictionary of optional kwargs passed through upon initialization
+        of pysat instrument. (default=None)
+
+    """
+
+    elements = list(kwargs['load'].keys())
```

Comment:
Will this not immediately break if kwargs is None?

Reply:
Dropping the default of None since we never want to run that case